### ISSUE TITLE
Run tests on PR

### ISSUE THREAD

**Issue by johng** [omnilaboratory/obd#6]
We should setup something like Travis CI to run the tests on PR

**Comment by mobilecheng** [comment]
Yes, we should use CI tools. Next, we need to complete test cases and try to use Travis CI.
Thank you, John.
